connector/protobuf/src/test/resources/protobuf/pyspark_test.desc

```diff
@@ -0,0 +1,7 @@
+
```

Comment:
Was this generated by something else? Should better have license header.

Reply:
@HyukjinKwon this is a binary file generated by the below commands(Protoc), though the .proto file has comments Protoc seems to strip down the comments. we can't manually edit the .desc file also. not sure what else options we have.

```
// protoc --java_out=connector/protobuf/src/test/resources/protobuf/ connector/protobuf/src/test/resources/protobuf/pyspark_test.proto
// protoc --descriptor_set_out=connector/protobuf/src/test/resources/protobuf/pyspark_test.desc --java_out=connector/protobuf/src/test/resources/protobuf/org/apache/spark/sql/protobuf/ connector/protobuf/src/test/resources/protobuf/pyspark_test.proto
```